### ISSUE TITLE
Swarmers can delete some shuttle walls

### ISSUE THREAD

**Issue by Slinky5890** [yogstation13/yogstation#1670]
http://imgur.com/a/Jl50v

Not hugely important, but still there.

**Comment by ShadowDeath6** [comment]
it's not adjacent to space in that picture is why, but that's a bug in its own right

**Comment by Slinky5890** [comment]
Yeah I understand that, I was meaning more about maybe somehow excluding only those walls from swarmer deconstructing or something.
